docs/troubleshoot/verify-fingerprint.md

```diff
@@ -32,14 +39,11 @@ When you transfer the PAX file between systems, you must use binary transfer mod
    - `RefRuntimeHash-1.13.0.txt` (text) 
    - `zowe-verify-authenticity.sh` (text)
 
-Each `RefRuntimeHash-v.r.m.txt` file is specific to a Zowe release, where `v.r.m` is your Zowe release number, e.g. `1.9.0`.  Further 
-`RefRuntimeHash-v.r.m.txt` files will be added to the PAX file as each release becomes available.  This includes releases after v1.14.  
-
+Each `RefRuntimeHash-V.v.p.txt` file is specific to a Zowe release, where `V.v.p` is your Zowe release number, for example, `1.9.0`. This list of files is updated to include new Zowe releases as they become available. For example, if you use Zowe version 1.14, you will see `RefRuntimeHash-1.14.0.txt` in the list. 
```

Comment:
The conventional IBM z/OS release nomenclature is VRMF, or v.r.m.f (version, release, modification, fix).  Your convention is new to me, but I don't mind it, provided it's an understood convention.

Reply:
@John-A-Davies yes, it used to be `v.r.m`. However, IIRC this was changed to `V.v.p` everywhere in the doc to match the semantic versioning scheme. See topics [Zowe releases](https://docs.zowe.org/stable/troubleshoot/troubleshoot-zowe-release.html#zowe-releases) and [Obtaining the Zowe convenience build](https://docs.zowe.org/stable/user-guide/install-zowe-zos-convenience-build.html#obtaining-and-preparing-the-convenience-build), which both used `V.v.p`.